### PR TITLE
Remove db related deployment logic

### PR DESCRIPTION
**Pull request by garettarrowood**
This is the second pass at removing all database logic from Playbook, as a followup to #307 . This is being opened as a second PR as all deletions deal specifically with helm deployment logic.

### CODE REVIEW

File: bin/deploy [hunk 1]
```diff
@@ -5,28 +5,6 @@ revision=$2
...
-environmentDbValuesFile=""
-if [ -e $(pwd)/config/deploy/$environment/values.db.yaml ]; then
-    environmentDbValuesFile="-f $(pwd)/config/deploy/$environment/values.db.yaml"
-fi
-
-# deploy postgresql with helm
-sops --decrypt $(pwd)/config/deploy/${environment}/secrets.db.yaml > $(pwd)/config/deploy/${environment}/secrets.db.dec.yaml
-helm upgrade --install --wait --timeout 600 \
-    playbook-db-${environment} stable/postgresql --version 3.9.1 \
-    --namespace playbook-${environment} --kube-context=${cluster} --tiller-namespace=app-tiller \
```
Comment (garettarrowood) [inline bin/deploy]: Perhaps some parts of this `helm` command need to stay?

File: bin/deploy [hunk 2]
```diff
@@ -5,28 +5,6 @@ revision=$2
...
-
-helm_exit=$?
-rm $(pwd)/config/deploy/${environment}/secrets.db.dec.yaml
-
-if [ $helm_exit != 0 ]; then
-  echo "helm deploy failed"
-  exit $helm_exit
-fi
-
 sops --decrypt $(pwd)/config/deploy/${environment}/secrets.yaml > $(pwd)/config/deploy/${environment}/secrets.dec.yaml
```
Comment (benlangfeld) [inline bin/deploy]: This is a reference to a file you deleted in this commit, right?

Comment (xjunior) [reply bin/deploy]: Actually it was secrets.db.yaml

Comment (benlangfeld) [reply bin/deploy]: No, my bad

**Comment by benlangfeld** [comment]
Once this ships, be sure to `helm delete --purge` the appropriate releases.

File: bin/deploy (hunk 1 above)
Comment (benlangfeld) [reply bin/deploy]: Why?

Comment (garettarrowood) [reply bin/deploy]: I guess not then :).

**Comment by garettarrowood** [comment]
Any chance this huddle-db pod failure would be resolved by the deletions in this PR? https://ci.powerhrg.com/blue/organizations/jenkins/powerhome%2Fplaybook/detail/master/183/pipeline

File: config/deploy/templates/db_migrate_hook.yaml.erb [hunk 3]
```diff
@@ -1,40 +0,0 @@
-apiVersion: v1
-kind: Pod
-metadata:
-  name: "huddle-db-migrate-<%= deployment_id %>"
```
Comment (garettarrowood) [inline config/deploy/templates/db_migrate_hook.yaml.erb]: Looks like this hook may need to be removed. Not sure why this didn't trigger in the PR's CI.

**Pull request merged**